modules/ROOT/pages/cbl-whatsnew.adoc

```diff
@@ -17,6 +17,39 @@ NOTE: Couchbase Lite 3.0 introduces some breaking changes. +
 If you are upgrading from 2.x, please refer to the appropriate upgrade page -- see: <<lbl-upgrade>>
 Users should be able to upgrade to 3.1.x from 3.0.x without manual intervention.
 
+== Release 3.2.0 - Beta 2 - Java Desktop (April 2024)
+
+=== New Features
+
+==== Vector Search
+
+[IMPORTANT] 
+--
+Vector Search is only available for 64-bit architectures and
```

Comment:
Small edit to change the placement of the word "only" (Pet peeve :-) ) Also, instead of "machine," should you say "device?"
Vector Search is available only on devices with 64-bit architectures and Intel processors that support the Advanced Vector Extensions 2 (AVX2) instruction set. To verify whether your device supports the AVX2 instruction set, follow these instructions.

Reply:
Done!